README.md

```diff
@@ -102,11 +102,50 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Create and run data jobs locally
+
+```
+pip quickstart-vdk
+```
+
+This installs the core vdk packages and the vdk command line interface. You can
+use them to run jobs in your local shell environment.
+
+See also the [Getting
+Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started)
+section of the wiki
+
+## Deploy and manage jobs in a cloud-like environment
+
+**Prerequisites**
+- [helm](https://helm.sh/docs/intro/install)
+- [docker](https://docs.docker.com/get-docker)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) (version 0.11.1 or later)
+
+```
+vdk server --install
+```
+
+The server includes:
+- control service - provides an API for managing the lifecycle of Data Jobs and
+  an execution environment where jobs can be deployed
+- operations ui - the vdk user interface
+- git server - stores data job source code
+- docker registry - stores data job images
+
+You can then use the vdk cli to create and deploy jobs and the UI to manage
+them.
+
+**Video tutorials on getting started with quickstart-vdk and the UI**
+
+- [Getting started with VDK Operations UI](https://www.youtube.com/watch?v=DLRGCCGUp0U)
+- [VDK Operations UI Overview](https://www.youtube.com/watch?v=9BkAOSvXuUg)
```

Comment:
I would rather look for the YouTube channel video tutorials in the Next Steps section. What do you think?